### PR TITLE
[Feature] Updated and add icons

### PR DESCRIPTION
**Pull request by aappoalander**
## Features
This PR adds the following:
- Illustrative icons highlight colour can be changed
- Fixes for illustrative icon colors
- New illustrative icons: registers
- New component icons: radioButton, radioButtonLarge
- New base icons: listBulleted, listNumbered, subDirectory, upload
- Support for prepare and automatic build when publishing
Breaking changes:
- Illustrative icons svg class names are no longer optimised and have changed.
- Component icon toggle: multiple internal changes, style updates

### CODE REVIEW

File: src/illustrativeIcons/icon-illustration-global.svg [hunk 1]
```diff
@@ -1,42 +1,38 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="80px" height="80px" viewBox="0 0 80 80" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: Sketch 49.3 (51167) - http://www.bohemiancoding.com/sketch -->
-    <title>Ikonit / Kuvitukset / Kansainvälistyminen</title>
-    <desc>Created with Sketch.</desc>
-    <defs></defs>
-    <g id="Ikonit-/-Kuvitukset-/-Kansainvälistyminen" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+    <title>Icons/Illustrative Icons/Internationalization</title>
+    <g id="Icons/Illustrative-Icons/Internationalization" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
         <g id="Kansainvälistyminen" transform="translate(8.000000, 8.000000)">
```
Comment (ketsappi) [inline src/illustrativeIcons/icon-illustration-global.svg:5]: Just a side note, but is it possible that Scandinavian characters might cause troubles in the svgs?

Comment (aappoalander) [reply src/illustrativeIcons/icon-illustration-global.svg:5]: Should not be possible. Uses utf-8 characterset.

Comment (aappoalander) [reply src/illustrativeIcons/icon-illustration-global.svg:5]: And those id's are never referred by anyone, but in many cases are not cleaned out by compilation.

**Review by LJKaski — APPROVED** [review]
Looks good to me and seems to work as intended.

**Review by ketsappi — APPROVED** [review]

**Pull request merged**